h2o-py/h2o/grid/grid_search.py

```diff
@@ -694,11 +706,33 @@ def gini(self, train=False, valid=False, xval=False):
         :param bool valid: If valid is True, then return the Gini Coefficient value for the validation data.
         :param bool xval:  If xval is True, then return the Gini Coefficient value for the cross validation data.
 
-        :returns: The Gini Coefficient for this binomial model.
+        :returns: The Gini Coefficient for the models in this grid.
         """
         return {model.model_id: model.gini(train, valid, xval) for model in self.models}
 
 
+    # @alias('pr_auc')
+    def aucpr(self, train=False, valid=False, xval=False):
+        """
+        Get the aucPR (Area Under PRECISION RECALL Curve).
+
+        If all are False (default), then return the training metric value.
+        If more than one options is set to True, then return a dictionary of metrics where the keys are "train",
+        "valid", and "xval".
+
+        :param bool train: If train is True, then return the aucpr value for the training data.
+        :param bool valid: If valid is True, then return the aucpr value for the validation data.
+        :param bool xval:  If xval is True, then return the aucpr value for the validation data.
+
+        :returns: The AUCPR for the models in this grid.
+        """
+        return {model.model_id: model.aucpr(train, valid, xval) for model in self.models}
+
+    @deprecated(replaced_by=aucpr)
+    def pr_auc(self):
+        pass
```

Comment:
after discussion with @ledell , we decided to deprecate legacy accessor for `pr_auc` and make `aucpr` the new standard on client side.
Alternative (also now available), was to make it an alias.

Reply:
@seb-h2o can you please run this by @NidhiMehta so that we know there is no issue for users? there were bunch of issues/confusion with aucpr recently

Reply:
will tell her: note that the `@deprecated` decorator, leaves the `pr_auc` method visible and functional.
It just raises a Py warning on call and then forwards to the `replaced_by` function if provided, so there shouldn't be any backwards incompatibilities.